apps/teamsjs-diagnostic-app/api/local.settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "IsEncrypted": false,
+    "Values": {
+      "FUNCTIONS_WORKER_RUNTIME": "node",
+      "AzureWebJobsFeatureFlags": "EnableWorkerIndexing",
+      "AzureWebJobsStorage": "UseDevelopmentStorage=true"
+    },
+    "Host": {
+        "CORS": "*"
```

Comment:
Why CORS is set to `*` do you think it's a security issues?

Reply:
I did some research and this is a potential security risk, so I can instead do `CORS": "http://localhost:3000" `